Makefile

```diff
@@ -89,3 +90,14 @@ uninstall: scripts_uninstall modules_uninstall
 lunatik_sym.h: $(LUA_API)
 	${shell ./gensymbols.sh $(LUA_API) > lunatik_sym.h}
 
+moontastik_install_%:
+	[ $* ] || (echo "Which target would you like to install?" && exit 1)
+	wget https://github.com/luainkernel/moontastik/releases/download/v0.1c/moontastik_lua.zip -O moontastik_lua.zip
```

Comment:
I would move the release version to a variable.. so it would be easier to adjust it later